src/zarr/testing/store.py

```diff
@@ -181,6 +181,23 @@ async def test_set_many(self, store: S) -> None:
         for k, v in store_dict.items():
             assert (await self.get(store, k)).to_bytes() == v.to_bytes()
 
+    @pytest.mark.parametrize("key", ["zarr.json", "c/0", "foo/c/0.0", "foo/0/0"])
+    async def test_set_partial_values(self, store: S, key: str) -> None:
+        """
+        Ensure that data can be written to the store using the store.set_partial_values method.
+        """
+        assert not store.read_only
+        # Create empty key
+        await store.set(key, self.buffer_cls.from_bytes(b""))
```

Comment:
`set_partial_values` for a `LocalStore` requires that a key exist. This test assumes this is the intended behavior and creates a key containing no data prior to testing set_partial_values.